flink-runtime/src/main/java/org/apache/flink/runtime/io/network/netty/NettyServer.java

```diff
@@ -198,6 +202,10 @@ private void initEpollBootstrap() {
         // multiple servers running on the same host.
         String name = NettyConfig.SERVER_THREAD_GROUP_NAME + " (" + config.getServerPort() + ")";
 
+        if (!Epoll.isAvailable()) {
```

Comment:
This seems unnecessary considering that the `EpollEventLoopGroup` already checks it.